string.c

```diff
@@ -8282,20 +8282,29 @@ rb_str_capitalize_bang(int argc, VALUE *argv, VALUE str)
 
 /*
  *  call-seq:
- *    capitalize(mapping) -> string
+ *    capitalize(mapping = :ascii) -> string
  *
- *  Returns a string containing the characters in +self+;
- *  the first character is upcased;
- *  the remaining characters are downcased:
+ *  Returns a string containing the characters in +self+,
+ *  each with possibly changed case:
  *
- *     s = 'hello World!' # => "hello World!"
- *     s.capitalize       # => "Hello world!"
+ *  - The first character is upcased.
+ *  - Other characters are downcased.
  *
- *  The casing may be affected by the given +mapping+;
- *  see {Case Mapping}[rdoc-ref:case_mapping.rdoc].
+ *  Examples:
+ *
+ *    'hello'.capitalize # => "Hello"
+ *    'HELLO'.capitalize # => "Hello"
```

Comment:
It should be clarified that the first character of the whole string, not each words, will be upcased, as the previous examples, I think.

Reply:
> It should be clarified that the first character of the whole string, not each words, will be upcased, as the previous examples, I think.

Thanks, @nobu.  I should have realized that.  Will fix.